app/reporting-framework/json-reports/ovc-patient-list-template.json

```diff
@@ -0,0 +1,181 @@
+            "join": {
+                "type": "LEFT",
+                "joinCondition": "fh.encounter_type = et.encounter_type_id"
+            }
+        },
+        {
+            "table": "amrs.person_address",
+            "alias": "pa",
+            "join": {
+                "type": "LEFT",
+                "joinCondition": "t1.person_id = pa.person_id"
+            }
+        },
+        {
+            "table": "amrs.patient_identifier",
+            "alias": "ovc_id",
+            "join": {
+                "type": "LEFT",
+                "joinCondition": "t1.person_id = ovc_id.patient_id AND (ovc_id.voided IS NULL || ovc_id.voided = 0) AND ovc_id.identifier_type = 43"
+            }
+        }
+
+    ],
+    "columns": [
+        {
+            "type": "simple_column",
+            "alias": "patient_uuid",
+            "column": "t1.uuid"
+        },
+        {
+            "type": "derived_column",
+            "alias": "uuid",
+            "expressionType": "simple_expression",
+            "expressionOptions": {
+                "expression": "t1.uuid"
+            }
+        },
+        {
+            "type": "simple_column",
+            "alias": "person_id",
+            "column": "t1.person_id"
+        },
+        {
+            "type": "derived_column",
+            "alias": "person_name",
+            "expressionType": "simple_expression",
+            "expressionOptions": {
+                "expression": " CONCAT(COALESCE(person_name.given_name, ''), ' ', COALESCE(person_name.middle_name, ''), ' ', COALESCE(person_name.family_name, ''))"
+            }
+        },
+        {
+            "type": "derived_column",
+            "alias": "identifiers",
+            "expressionType": "simple_expression",
+            "expressionOptions": {
+                "expression": " GROUP_CONCAT(DISTINCT id.identifier SEPARATOR ', ')"
+            }
+        },
+        {
+            "type": "derived_column",
+            "alias": "phone_number",
+            "expressionType": "simple_expression",
+            "expressionOptions": {
+                "expression": " GROUP_CONCAT(DISTINCT contacts.value SEPARATOR ', ')"
+            }
+        },
+        {
+            "type": "simple_column",
+            "alias": "latest_rtc_date",
+            "column": "date_format(fh.rtc_date, '%Y-%m-%d')"
+        },
+        {
+            "type": "simple_column",
+            "alias": "latest_vl",
+            "column": "fh.vl_1"
+        },
+        {
+            "type": "simple_column",
+            "alias": "latest_vl_date",
+            "column": "date_format(fh.vl_1_date, '%Y-%m-%d')"
+        },
+        {
+            "type": "simple_column",
+            "alias": "last_appointment",
+            "column": "CONCAT(COALESCE(DATE_FORMAT(fh.encounter_datetime, '%Y-%m-%d'),''),' ',COALESCE(et.name, ''))"
+        },
+        {
+            "type": "simple_column",
+            "alias": "cur_meds",
+            "column": "fh.cur_arv_meds"
```

Comment:
remove cur_meds and cur_arv_line, previous_vl,  previous_vl_date , unless they ask for it